pkg/operator/render.go

```diff
@@ -14,7 +14,6 @@ import (
 	"github.com/golang/glog"
 
 	configv1 "github.com/openshift/api/config/v1"
-
 	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
```

Comment:
can you revert this change as well since it's an autospacing?

Reply:
done